File: one.cpp
#include <bits/stdc++.h>
using ll = long long;
using namespace std;

void one() {
  ifstream fin("one.txt");
  vector<int> one, two;
  int a, b;
  while (!fin.eof()) {
    fin >> a >> b;
    one.push_back(a);
    two.push_back(b);
  }
  sort(one.begin(), one.end());
  sort(two.begin(), two.end());
  ll sum = 0;
  for (int i = 0; i < one.size(); i++) {
    sum += abs(one[i] - two[i]);
  }
  cout << sum << "\n";
}

void two() {
  ifstream fin("one.txt");
  vector<int> one;
  multiset<int> two;
  int a, b;
  while (!fin.eof()) {
    fin >> a >> b;
    one.push_back(a);
    two.insert(b);
  }
  ll sum = 0;
  for (int i = 0; i < one.size(); i++) {
    sum += one[i] * two.count(one[i]);
  }
  cout << sum << "\n";
}

int main() {
  one();
  two();
}

File: one.py
ones, twos = [], []
with open("one.txt", "r") as read:
	for line in read:
		one, two = map(int, line.split())
		ones.append(one)
		twos.append(two)
ones.sort()
twos.sort()

def one():
    print(sum(abs(one - two) for one, two in zip(ones, twos)))

def two():
    print(sum(one * twos.count(one) for one in ones))

one()
two()